draft-ietf-ohai-ohttp.md

```diff
@@ -129,17 +130,22 @@ to correlate behavior. This imposes considerable performance and efficiency over
 to the additional round trip to the server (at a minumum), additional data exchanged, and
 additional CPU cost of cryptographic computations.
 
-This document defines two kinds of HTTP resources -- Oblivious Relay Resources
-and Oblivious Gateway Resources -- that process encapsulated binary HTTP messages
-{{BINARY}} using Hybrid Public Key Encryption (HPKE; {{!HPKE=RFC9180}}). They can be composed to
-protect the content of encapsulated requests and responses, thereby separating the identity of a
-requester from the request.
+To overcome these limitations, this document defines how encapsulated binary
```

Comment:
```suggestion
To overcome these limitations, this document defines how binary
```
HPKE is the thing that encapsulates the messages, it doesn't take encapsulated messages as input.